stylesheets/styles.css

```diff
@@ -166,15 +166,20 @@ body, html {
 
     .map {
         margin-top: 1em;
-        width: 100%;
-        height: 85%;
+        width: 50%;
+        height: 90%;
+        border-radius: 5px;
     }
```

Comment:
Maybe change border-radius from 'px' to 'em' units to keep the stylesheet uniform?